### PR TITLE
Use connection key for global webhook key

### PR DESCRIPTION
**Pull request by bryce-fitzsimons**
### Features and Changes

Global webhooks were using a hardcoded key from the environment variable when they should be using a dynamic key from the SDK connection (to match how SDK webhooks work).

Also fixed a spelling error in one of the related types + migration

### CODE REVIEW

**Review by gazzdingo — APPROVED** [review]
LGTM

File: packages/back-end/src/models/SdkWebhookLogModel.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { randomUUID } from "crypto";
 import omit from "lodash/omit";
 import mongoose from "mongoose";
-import { migrateSdkWebhookLogModel } from "@back-end/src/util/migrations";
+import { migrateSdkWebhookLogModel } from "../util/migrations";
```
Comment (gazzdingo) [inline packages/back-end/src/models/SdkWebhookLogModel.ts:4]: should that be @back-end/src/util/migrations
not ...

File: packages/back-end/src/util/migrations.ts [hunk 2]
```diff
@@ -736,3 +737,13 @@ export function migrateSavedGroup(
 
   return group;
 }
+
+export function migrateSdkWebhookLogModel(
+  doc: SdkWebHookLogDocument
+): SdkWebHookLogDocument {
+  if (doc?.webhookRequestId) {
```
Comment (jdorn) [inline packages/back-end/src/util/migrations.ts:744]: Typo?

Comment (bryce-fitzsimons) [inline packages/back-end/src/util/migrations.ts:744]:
```suggestion
  if (doc?.webhookReduestId) {
```

Comment (bryce-fitzsimons) [reply packages/back-end/src/util/migrations.ts]: ha, yes! fixed

**Pull request merged**